check_os.py

```diff
@@ -28,7 +28,8 @@ def _do_popup(title, text):
         import win32api
         win32api.MessageBox(0, text, title)
     except ImportError:
-        os.system('xmessage -center "$(printf "%s")"' % (text))
+        import subprocess
+        subprocess.Popen(['xmessage', '-center', text], shell=False)
```

Comment:
Was this tested with newlines in the text?

Reply:
Just checked it, totally works with newlines.